testlib/src/lib.rs

```diff
@@ -38,3 +38,14 @@ pub fn with_kernel_module<F: Fn()>(f: F) {
     let _m = LoadedModule::load(env::var("KERNEL_MODULE").unwrap());
     f();
 }
+
+pub fn assert_dmesg_contains(msgs: &[&[u8]]) {
+    let output = Command::new("dmesg").output().unwrap();
```

Comment:
assert returned zero? do we care?

Reply:
Was just a copy paste from the existing function. I'll do a fix in a follow up PR if that works?

Reply:
No strong opinions, I mostly just realized from the recent build.rs change that Rust doesn't work like `check_output` and now I'm worried about it :)